### ISSUE TITLE
User can create empty decks in Web UI

### ISSUE THREAD

**Issue by spiralsix** [spiralsix/flashcards#3]
They have must have Name, Description and can have tags

**Comment by bacitracin** [comment]
Would there be empty cards associated with this deck as well?

**Comment by spiralsix** [comment]
No.  Just decks which don't yet have cards.  Question and answer are
required for cards
